specification/deviceupdate/resource-manager/Microsoft.DeviceUpdate/preview/2020-03-01-preview/examples/Accounts/Accounts_Create.json

```diff
@@ -18,7 +18,8 @@
         "type": "Microsoft.DeviceUpdate/accounts",
         "properties": {
           "provisioningState": "Accepted",
-          "hostName": "contoso.api.adu.microsoft.com"
+          "hostName": "contoso.api.adu.microsoft.com",
+          "privateEndpointConnections": []
```

Comment:
That would be great if you could add sample detail for `privateEndpointConnections` that will benefit the end user in document.

Reply:
Added PrivateEndpointConnection payload to Account examples (except Create, PEC is not created by customers but by us when Network calls us when they are creating their own).